views/news/tabs.js

```diff
@@ -14,15 +15,11 @@ export default [
     },
   },
   {
-    id: 'PoliticOleNewsView',
-    title: 'PoliticOle',
-    rnVectorIcon: {iconName: 'megaphone'},
-    component: NewsContainer,
-    props: {
-      url: 'http://oleville.com/politicole/feed/',
-      mode: 'rss',
-      name: 'PoliticOle',
-    },
+    id: 'OlevilleView',
+    title: 'Oleville',
+    rnVectorIcon: {iconName: 'ion-mouse'},
```

Comment:
Sadly, it has to be an icon that's available in an `ios-` variant.

See https://github.com/StoDevX/AAO-React-Native/blob/master/views/components/tabbed-view/index.ios.js#L35 – the tab bar loads `ios-${name}-outline` and `ios-${name}` variants.

You also don't need the `ion-` prefix, as rn-vector-icons removes that.